packages/workflow/src/records/fhir.ts

```diff
@@ -300,6 +300,48 @@ export function createCorrectedTask(
   }
 }
 
+export function createRejectTask(
+  previousTask: Task,
+  practitioner: Practitioner,
+  statusReason: fhir3.CodeableConcept
+): Task {
+  return {
```

Comment:
couldn't we have just spread the previous task and then replace the values that we want to replace?